### PR TITLE
chore(action): Replaced semantic PR with an action version

### PR DESCRIPTION
**Pull request by greg-in-a-box**
Replaced existing Semantic Pull Request GH app config with Semantic Pull Request GH action config

ref: https://github.com/amannn/action-semantic-pull-request

### CODE REVIEW

File: .github/workflows/semantic.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            - opened
+            - edited
+            - synchronize
+
+jobs:
+    main:
+        name: Validate
+        runs-on: ubuntu-latest
+        steps:
+            - uses: amannn/action-semantic-pull-request@v4
```
[the remark this answers is not in the document]
Comment (greg-in-a-box) [reply .github/workflows/semantic.yml:15]: Good question, Is there a process for that ?

**Review by tjuanitas — APPROVED** [review]
LGTM. Do we have to set up any sort of configuration settings?

File: .github/semantic.yml [hunk 2]
```diff
@@ -1,2 +0,0 @@
-titleAndCommits: true
-anyCommit: true
```
Comment (tjuanitas) [inline .github/semantic.yml:2]: Do we need to add these settings for the GH Action?

Comment (greg-in-a-box) [reply .github/semantic.yml:2]: Not that I am aware of, should work out of the box with the configuration I have in there

**Pull request merged**